ghost/core/core/server/services/settings/SettingsBREADService.js

```diff
@@ -360,13 +360,6 @@ class SettingsBREADService {
      * @private
      */
     async sendEmailVerificationMagicLink({email, key}) {
-        const [,toDomain] = email.split('@');
-
-        let fromEmail = `noreply@${toDomain}`;
-        if (fromEmail === email) {
-            fromEmail = `no-reply@${toDomain}`;
-        }
-
```

Comment:
So the reason we added this particular logic was to avoid both `to` and `from` address to be the same value, which makes Email Providers to mark the email as spam or Gmail shows a yellow banner. So in case the to address is same as from `noreply@domain`, we change the sender to value to `no-reply` to avoid that.